lib/converter.js

```diff
@@ -107,6 +107,9 @@ async function convert (processor, inputFile, options, timings, watch, preview)
         pdfOptions.format = format // Paper format. If set, takes priority over width or height options. Defaults to 'Letter'.
       }
 
+      // set the puppeteer render timeout (https://github.com/puppeteer/puppeteer/blob/main/docs/api.md#pagepdfoptions)
+      pdfOptions.timeout = browserInstance.renderingTimeout;
```

Comment:
We should configure a specific timeout, see: https://github.com/Mogztter/asciidoctor-web-pdf/issues/612#issuecomment-1028033740
We can also set this value when we declare/initialize the `pdfOptions` object.